INN/umbrella-inndev
# Split up grid-column-start/end declaration

## Changes

This pull request makes the following changes:

- Splits up the `grid-column: 1 /\ -1;` css to be 
```
grid-column-start: 1;
grid-column-end: -1;
```
since LESS didn't compile it correctly and instead of coming out as `grid-column: 1 / -1` it compiles as `grid-column: 1 / \ -1;`, resulting in a wonky grid display

## Why



For #127 ish

## Testing/Questions

Features that this PR affects:

- Grid column display for 2/3 rows


Questions that need to be answered before merging:


Steps to test this PR:

1. View the embed with 2 rows and then 3 rows and make sure they look as expected
